package.json

```diff
@@ -8,23 +8,24 @@
     "react": "^16.8.6",
     "react-bootstrap": "^1.0.0-beta.8",
     "react-dom": "^16.8.6",
-    "react-structural-metadata-editor": "https://github.com/avalonmediasystem/react-structural-metadata-editor#avalon-r6.5",
+    "react-structural-metadata-editor": "https://github.com/avalonmediasystem/react-structural-metadata-editor",
```

Comment:
Remove version pin for the Structural Metadata Editor (SME). Upstream doesn't follow semver release tagging. yarn.lock pins to a specific commit.